solana/program_v2/programs/gateway_v2/src/util.rs

```diff
@@ -105,6 +107,17 @@ pub fn check_gatekeeper_auth_threshold(auth_keys: &[AuthKey], auth_threshold: u8
     auth_key_count >= auth_threshold as usize
 }
 
+pub fn check_network_auth_threshold(auth_keys: &[AuthKey], auth_threshold: u8) -> bool {
```

Comment:
Should this also rather be a can_access? in a trait or state object implementation?

Reply:
Looks like this was done will remove this function